webapp/src/components/App/Singlestudy/explore/Modelization/Areas/Storages/StorageConfig/AdditionalConstraints/AddConstraintDialog.tsx

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright (c) 2025, RTE (https://www.rte-france.com)
+ *
+ * See AUTHORS.txt
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This file is part of the Antares project.
+ */
+
+import AddCircleIcon from "@mui/icons-material/AddCircle";
+import { useMemo } from "react";
+import { useTranslation } from "react-i18next";
+import FormDialog from "@/components/common/dialogs/FormDialog";
+import Fieldset from "@/components/common/Fieldset";
+import type { SubmitHandlerPlus } from "@/components/common/Form/types";
+import SelectFE from "@/components/common/fieldEditors/SelectFE";
+import StringFE from "@/components/common/fieldEditors/StringFE";
+import SwitchFE from "@/components/common/fieldEditors/SwitchFE";
+import { createAdditionalConstraints } from "@/services/api/studies/areas/storages";
+import type { AdditionalConstraint } from "@/services/api/studies/areas/storages/types";
+import { validateString } from "@/utils/validation/string";
+import { CONSTRAINT_OPERATORS, CONSTRAINT_VARIABLES, DEFAULT_CONSTRAINT_VALUES } from "./constants";
+
+interface Props {
+  open: boolean;
+  onClose: () => void;
+  onSave: () => void;
+  studyId: string;
+  areaId: string;
+  storageId: string;
+}
+
+interface FormValues {
+  name: string;
+  variable: AdditionalConstraint["variable"];
+  bounds: AdditionalConstraint["operator"];
```

Comment:
More clear to use 'operator', 'bounds' is used for the label, there is no link with the code